src/ace/ari_text/parsemod.py

```diff
@@ -155,12 +172,20 @@ def p_reportlist_join(p):
 def p_reportlist_end(p):
     'reportlist : report'
     p[0] = [p[1]]
-
-
+    
 def p_report(p):
-    'report : LPAREN VALSEG EQ VALSEG SC VALSEG EQ ari SC acbracket RPAREN'
-    rawtime = util.TYPEDLIT[StructType.TD](p[4])
-    rel_time = BUILTINS_BY_ENUM[StructType.TD].convert(LiteralARI(rawtime, StructType.TD))
+    '''report : LPAREN VALSEG EQ VALSEG SC VALSEG EQ ari SC acbracket RPAREN 
+              | LPAREN VALSEG EQ typedlit SC VALSEG EQ ari SC acbracket RPAREN '''
```

Comment:
This could be simplified, I think, so both parts being `VALSEG EQ ari` and check the ARI type in the function.